src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -1830,7 +1830,7 @@ public unsafe void PutSecretKey(
 
             var tree = context.Transaction.InnerTransaction.CreateTree("SecretKeys");
 
-            if (overwrite == false && tree.Read(name) != null)
+            if (overwrite == false && !tree.Read(name).HasValue == false)
```

Comment:
:)

Reply:
What a sneaky one! I thought I found them all. Will fix.

Reply:
Fixed. Fortunately there were just a few abominations like this.